### PR TITLE
Explicitly set expose function code action to preferred

### PR DESCRIPTION
**Pull request by Razzeee**
Don't default to preferred = true for all codeActions

### CODE REVIEW

**Comment by jmbockhorst** [comment]
I went ahead and implemented a code actions preference system like VSCode Typescript has, so we can manage it better.

**Comment by Razzeee** [comment]
Yeah, I actually agree, that should be the top choice in the future too

File: src/providers/codeAction/exposeUnexposeCodeAction.ts [hunk 1]
```diff
@@ -39,6 +42,7 @@ CodeActionProvider.registerRefactorAction(refactorName, {
       } else {
         result.push({
           title: "Expose Function",
+          isPreferred: true,
```
Comment (Razzeee) [inline src/providers/codeAction/exposeUnexposeCodeAction.ts:45]: This seems fishy to me. Should `IRefactorCodeAction` also start to extend ICodeAction? Should we omit `isPreferred` from that interface?

Comment (jmbockhorst) [reply src/providers/codeAction/exposeUnexposeCodeAction.ts:45]: Yeah, that’s a good idea. I just forgot to remove it here.

Comment (Razzeee) [reply src/providers/codeAction/exposeUnexposeCodeAction.ts]: Done

**Pull request merged**